packages/styles/scss/ibm-dotcom-styles.scss

```diff
@@ -26,3 +26,4 @@
 @import 'patterns/leadspace/leadspace';
 @import 'patterns/listsection/listsection';
 @import 'patterns/simplelongform/simplelongform';
+@import 'patterns/tableofcontents/tableofcontents';
```

Comment:
@sannminwin isn't this under `components` now?

Reply:
updated.